CorsixTH/Lua/humanoid_actions/seek_reception.lua

```diff
@@ -23,6 +23,9 @@ class "SeekReceptionAction" (HumanoidAction)
 ---@type SeekReceptionAction
 local SeekReceptionAction = _G["SeekReceptionAction"]
 
+-- distance from queue that patient can join a reception desk queue
+local can_join_queue_distance = 6
```

Comment:
Looking at room.lua and app.lua there are hints this variable should be ALL_CAPS...
Though the coding conventions do not give strict guidance to support this. @TheCycoONE ?

Reply:
We're not consistent, see e.g. hours_per_day in date.lua or case_pattern in strings.lua. I don't feel strongly about it and I'm not going to push the matter either way. (In C++ we reserve screaming snake case exclusively for preprocessor macros, but in lua there's no such considerations.)